packages/mui-lab/src/LoadingButton/LoadingButton.test.js

```diff
@@ -72,4 +73,44 @@ describe('<LoadingButton />', () => {
       expect(screen.getByRole('button')).to.have.text('loading…Test');
     });
   });
+
+  describe('ButtonGroup works with LoadingButton', () => {
+    it('correctly passes props to children', () => {
+      const { getByRole } = render(
+        <ButtonGroup variant="contained" size="large" color="secondary">
+          <LoadingButton />
+        </ButtonGroup>,
+      );
+      const button = getByRole('button');
+      expect(button).to.have.class(buttonClasses.contained);
+      expect(button).to.have.class(buttonClasses.sizeLarge);
+      expect(button).to.have.class(buttonClasses.containedSecondary);
+    });
+
+    it('correctly applies position classes to buttons', () => {
```

Comment:
Let's duplicate this test with showing three loading button, to validate it works in this case too.

Reply:
I had forgotten to change the Buttons to LoadingButtons.